app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -226,70 +227,77 @@ private void downloadMedia(Media m) {
         req.allowScanningByMediaScanner();
         req.setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
         PermissionUtils.checkPermissionsAndPerformAction(getActivity(), WRITE_EXTERNAL_STORAGE,
-            () -> enqueueRequest(req), () -> Toast.makeText(getContext(),
-                R.string.download_failed_we_cannot_download_the_file_without_storage_permission,
-                Toast.LENGTH_SHORT).show(), R.string.storage_permission,
-            R.string.write_storage_permission_rationale);
+                () -> enqueueRequest(req), () -> Toast.makeText(getContext(),
+                        R.string.download_failed_we_cannot_download_the_file_without_storage_permission,
+                        Toast.LENGTH_SHORT).show(), R.string.storage_permission,
+                R.string.write_storage_permission_rationale);
 
```

Comment:
Can you make sure just required lines are edited, otherwise it is hard to review your PR.

Reply:
public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
        if (!editable ) {// Disable menu options for editable views
            if ( provider.getTotalMediaCount()!=0)
            {…….}
            else
            {
                getActivity().startActivity(getActivity().getIntent());
                getActivity().finish();
            }

Reply:
For such complicated edits, it is not a good idea to use github interface. Instead you make your changes on the local branch you worked, and after committing your changes push your changes. Pull request will be updated automatically.

Please make sure you followed the contributor documentation.